packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -236,4 +261,86 @@ export default class PaypalCommerceRatepayPaymentStrategy implements PaymentStra
         script.src = 'https://c.paypal.com/da/r/fb.js';
         document.body.appendChild(script);
     }
+
+    private async reinitializeStrategy(
+        options: PaymentInitializeOptions &
+            WithPayPalCommerceAlternativeMethodsPaymentInitializeOptions,
+    ): Promise<void> {
+        await this.deinitialize();
+        await this.initialize(options);
+    }
+
+    /**
+     *
+     * Polling mechanism
+     *
+     *
+     * */
+    private async initializePollingMechanism(
+        methodId: string,
+        callback: any,
+        gatewayId?: string,
+        paypalRatePayOptions?: PaypalCommerceRatePay,
+    ): Promise<void> {
+        await new Promise<void>((resolve, reject) => {
+            const timeout = setTimeout(resolve, this.pollingInterval);
+
+            this.stopPolling = () => {
+                clearTimeout(timeout);
+                reject();
+            };
+        });
+
+        try {
+            this.pollingTimer += this.pollingInterval;
+
+            const orderStatus = await this.paypalCommerceIntegrationService.getOrderStatus(
+                'paypalcommercealternativemethods',
```

Comment:
Is this a correct methodId?

Reply:
if you mean `paypalcommercealternativemethods` - yes, that's what BE expect to recieve